File: core/src/net/natruid/jungle/components/render/AngleComponent.java
package net.natruid.jungle.components.render;

import com.artemis.Component;
import com.artemis.annotations.PooledWeaver;

@PooledWeaver
public class AngleComponent extends Component {
    public static final AngleComponent NONE = new AngleComponent();

    public float rotation = 0;
}

File: core/src/net/natruid/jungle/components/render/AngleComponent.kt
package net.natruid.jungle.components.render

import com.artemis.PooledComponent

class AngleComponent : PooledComponent() {
    companion object {
        val NONE = AngleComponent()
    }

    var rotation = 0f

    override fun reset() {
        rotation = 0f
    }
}
